config/maintainers.json

```diff
@@ -43,13 +43,13 @@
     },
     {
       "github_username": "jvarness",
-      "alumnus": false,
+      "alumnus": true,
       "show_on_website": true,
       "name": "Jake Varness",
       "link_text": "dev.to",
-      "link_url": "https://dev.to/jvarness",
+      "link_url": "https://jvarness.blog/",
       "avatar_url": "https://avatars1.githubusercontent.com/u/6510879?s=460\u0026v=460",
-      "bio": "Team Lead and Senior Software Engineer at Cerner Corporation, husband, father, nerd"
```

Comment:
Might not be required to remove your bio to mark you as alumnus unless you want to.

Reply:
I would prefer to for now :)